### PR TITLE
Add Modal and Modal's Backdrop transition duration

### PR DESCRIPTION
**Pull request by frederikhors**

### CODE REVIEW

**Comment by frederikhors** [comment]
@bestguy, can you please merge and release?

**Comment by bestguy** [comment]
Thanks @frederikhors , will review and merge soon

**Comment by frederikhors** [comment]
> Thanks @frederikhors , will review and merge soon

@bestguy, can you give me merge permission please?

File: src/Modal.svelte [hunk 1]
```diff
@@ -34,7 +36,6 @@
   export let backdropClassName = '';
   export let contentClassName = '';
   export let external = undefined;
-  export let fade = true;
```
Comment (bestguy) [inline src/Modal.svelte]: We were aligning with Reactstrap's approach that sets fade on/off via a boolean prop (https://reactstrap.github.io/components/modals/).
Could you keep this behavior and add the duration props?

Comment (frederikhors) [reply src/Modal.svelte]: Watch now. Do you like?

**Pull request merged**